### ISSUE TITLE
[P3.2][Sourcing][Accessibility][Pretest][Multi combobox] Although there are no values, the listbox for the values is present in the source code. It is empty and has no label

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10825]
#### Is this a bug, enhancement, or feature request?

Bug

#### Describe your proposal.
When there are no values selected in multi combobox , still the div having role - listbox is displayed
This is causing issue for disabled users 

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular: 15
core platform: 0.43.14

**Comment by nikolay-kolarov** [comment]
Hi,

Yes, you can put aria-hidden="true", when it is empty.
Please check this implementation as a reference: https://ui5.sap.com/#/entity/sap.m.MultiComboBox/sample/sap.m.sample.MultiComboBox

Best Regards,
Nikolay
